Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:14.04 
```

Comment:
@fearphage I think the last time I tried to base an image on alpine I gave up because of a lack of package manager. Also, cool tip on the using `ADD` instead of `curl`! Unfortunately I rebase my last commit to this in and it looks like it blew away your comment on what you suggested. I'm surprised Github even removed it from the flow of this comments on this PR :(.